### PR TITLE
fix(resty): upgraded gosdk for restry

### PR DESCRIPTION
**Pull request by cnlangzi**
### Changes
- upgraded gosdk

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/core/transaction/http.go [hunk 1]
```diff
@@ -105,10 +105,15 @@ func makeSCRestAPICall(scAddress string, relativePath string, params map[string]
 		"Access-Control-Allow-Origin": "*",
 	}
 
+	options := []resty.Option{
+		resty.WithTransport(transport),
```
Comment (peterlimg) [inline code/go/0chain.net/core/transaction/http.go]: If the transport is the same as the default transport that is used in restry.New(), then we don't have to use it here again.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/core/transaction/http.go (hunk 1 above)
Comment (cnlangzi) [reply code/go/0chain.net/core/transaction/http.go]: fixed

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**